### PR TITLE
[microTVM]Enable TVMC micro with AoT Executor 

### PR DESCRIPTION
**Pull request by mehrdadh**
This PR enables AoT Executor for tvmc micro compilation.

### CODE REVIEW

**Review by leandron — COMMENTED** [review]
I know this is a draft, just was curious and ended up having a look...

File: gallery/how_to/work_with_microtvm/micro_tvmc.sh [hunk 1]
```diff
@@ -126,7 +124,7 @@ tvmc micro create \
     project \
     model.tar \
     zephyr \
-    --project-option project_type=host_driven zephyr_board=qemu_x86
+    --project-option project_type=host_driven board=qemu_x86
 # bash
 # This will generate a ``Host-Driven`` Zephyr project for ``qemu_x86`` Zephyr board. In Host-Driven template project,
 # the Graph Executor will run on host and perform the model execution on Zephyr device by issuing commands to the
```
[the remark this answers is not in the document]
Comment (mehrdadh) [reply gallery/how_to/work_with_microtvm/micro_tvmc.sh:130]: I think ideally the default for `tvmc compile` for `micro` should be using AoT executor. Graph Executor is becoming less important, specially for micro devices, as we improved AoT flow, so I suggest to just briefly mention the options.

Regarding the default AoT flow for `tvm compile` for micro devices, I haven't found a way yet. One possible way it to set it based on the runtime=crt. wdyt?

**Comment by mehrdadh** [comment]
@leandron friendly reminder about this PR.

**Review by alanmacd — APPROVED** [review]
LGTM

File: gallery/how_to/work_with_microtvm/micro_tvmc.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (mehrdadh) [reply gallery/how_to/work_with_microtvm/micro_tvmc.sh]: addressed in https://github.com/apache/tvm/pull/14077/commits/cf9a64f543e456e897f562570fa01b8cd7918b83

**Review by areusch — APPROVED** [review]
thanks @mehrdadh ! I think @leandron 's comments are addressed, so we can land this. Leandro, let us know if you have more feedback and we can address in follow-ups.

**Pull request merged**